main-service/src/main/java/ru/practicum/event/controller/EventController.java

```diff
@@ -0,0 +1,86 @@
+package ru.practicum.event.controller;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import ru.practicum.event.dto.*;
+import ru.practicum.event.service.EventService;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping
+public class EventController {
```

Comment:
нужно добавить Validated

```suggestion
@Validated
public class EventController {
```